scripts/bert/finetune_squad.py

```diff
@@ -0,0 +1,492 @@
+"""
+SQuAD with Bidirectional Encoder Representations from Transformers
+
+=========================================================================================
+
+This example shows how to implement finetune a model with pre-trained BERT parameters for
+SQuAD, with Gluon NLP Toolkit.
+
+@article{devlin2018bert,
+  title={BERT: Pre-training of Deep Bidirectional Transformers for Language Understanding},
+  author={Devlin, Jacob and Chang, Ming- \
+      Wei and Lee, Kenton and Toutanova, Kristina},
+  journal={arXiv preprint arXiv:1810.04805},
+  year={2018}
+}
+"""
+
+# coding=utf-8
+
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+# pylint:disable=redefined-outer-name,logging-format-interpolation
+
+import argparse
+import collections
+import json
+import logging
+import os
+import random
+import time
+import warnings
+
+import numpy as np
+import mxnet as mx
+from mxnet import gluon, nd
+
+import gluonnlp as nlp
+from gluonnlp.data import SQuAD
+from bert_qa_model import BertForQALoss, BertForQA
+from bert_qa_dataset import (SQuADTransform, preprocess_dataset)
+from bert_qa_evaluate import get_F1_EM, predictions
+
+np.random.seed(6)
+random.seed(6)
+mx.random.seed(6)
+
+log = logging.getLogger('gluonnlp')
+log.setLevel(logging.DEBUG)
+formatter = logging.Formatter(
+    fmt='%(levelname)s:%(name)s:%(asctime)s %(message)s', datefmt='%H:%M:%S')
+
+
+parser = argparse.ArgumentParser(description='BERT QA example.'
+                                 'We fine-tune the BERT model on SQuAD dataset.')
+
+parser.add_argument('--only_predict',
+                    action='store_true',
+                    help='Whether to predict only.')
+
+parser.add_argument('--model_parameters',
+                    type=str,
+                    default=None,
+                    help='Model parameter file')
```

Comment:
Could you also provide the option to load pretrained bert parameters like in https://github.com/dmlc/gluon-nlp/pull/571/ ?